backend/src/routes/notificationRoute.ts

```diff
@@ -1,9 +1,10 @@
 import express from "express";
-import { getNotification } from '../controllers/notificationController';
+import { getNotification, updateNotification } from '../controllers/notificationController';
 import { authorizeUser } from "../middleware/authMiddleware";
 
 const notificationRoute = express.Router()
 
 notificationRoute.get('/', authorizeUser, getNotification)
+notificationRoute.put('/', authorizeUser, updateNotification)
```

Comment:
```suggestion
notificationRoute.put('/:id', authorizeUser, updateNotification)
```

It will be simple like this. 

We have the same format like others update request.